app/pages/corrections/VatCorrectionsListPage.scala

```diff
@@ -33,7 +33,12 @@ case class VatCorrectionsListPage(periodIndex: Index) extends Page {
         case None => routes.JourneyRecoveryController.onPageLoad()
       }
     } else {
-      correctionRoutes.VatPeriodCorrectionsListController.onPageLoad(mode, answers.period)
+      if(mode == CheckThirdLoopMode){
+        correctionRoutes.VatPeriodCorrectionsListController.onPageLoad(NormalMode, answers.period)
+      }else{
```

Comment:
tiny one here but if making other changes can we add spaces before and after else 👍🏻

Reply:
theres quite a few places this applies, I wont comment them all

Reply:
done